cli/server/server.go

```diff
@@ -119,6 +120,7 @@ func newGraceContext() context.Context {
 	ctx, cancel := context.WithCancel(context.Background())
 	stop := make(chan os.Signal, 1)
 	signal.Notify(stop, os.Interrupt)
+	signal.Notify(stop, syscall.SIGTERM)
```

Comment:
Version for windows?

Reply:
Not needed, it has `syscall.SIGTERM` definition. It won't work there of course, but it compiles fine.